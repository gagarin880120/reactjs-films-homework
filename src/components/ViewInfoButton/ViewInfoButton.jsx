import React from 'react';
import styles from './ViewInfoButton.module.scss';

export default function ViewInfoButton() {
  return (
    <button className={styles.infoButton}>View Info</button>
  )
}
