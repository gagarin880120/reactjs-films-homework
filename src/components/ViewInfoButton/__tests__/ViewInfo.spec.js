import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import ViewInfoButton from '../ViewInfoButton';

test('ViewInfoButton component should render without crashing', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<ViewInfoButton />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
