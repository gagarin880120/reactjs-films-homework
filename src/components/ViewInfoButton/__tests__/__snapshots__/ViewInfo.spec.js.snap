// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewInfoButton component should render without crashing 1`] = `
<button
  className="infoButton"
>
  View Info
</button>
`;
